src/mvItemRegistry.cpp

```diff
@@ -1182,7 +1182,9 @@ DelaySearch(mvItemRegistry& registry, mvAppItem* item)
 mvAppItem* 
 GetItem(mvItemRegistry& registry, mvUUID uuid)
 {
-
+    // TODO: This is an improvement in performance, but actually it's been put here because otherwise in "check cache" it crashes -> get informed
+    if (uuid == 0)
+        return nullptr;
```

Comment:
Adding this as a quickpath is fine, but the comment seems to be a bit misleading. I couldn't recreate a crash on UUID 0.

```py
import dearpygui.dearpygui as dpg

dpg.create_context()
dpg.create_viewport(title="Test", width=700, height=750)

with dpg.window() as wnd:
    dpg.add_button(label="Click me", callback=lambda: dpg.configure_item(0, show=False))

# dpg.configure_item(0, show=False)

dpg.setup_dearpygui()
dpg.show_viewport()
dpg.start_dearpygui()
dpg.destroy_context()
```

Neither of the `configure_item` calls segfaults (or did you mean a Python exception and not a segfault?) - I tested both in old and in new versions, and they produce identical messages.

In fact, whenever the code finds a zero in `cachedContainersID`, the corresponding `cachedContainersPTR` contains null, keeping it from the initialization in `mvItemRegistry` constructor. This can be broken by calling `CacheItem` and passing it an item whose `uuid` field is zero - not sure whether such code path exists; I couldn't quickly find one. Looks like `CacheItem` is only called in places where the item already has a nonzero `uuid`.

That said, the comment probably needs correction.

Thinking more about it, how often does `GetItem` get called with UUID of zero? Won't that extra check just slow down most calls for no visible benefit? (sure, the slowdown will be negligible because `GetItem` is already slow as hell... still it looks to me like degrading 99.99% cases for the benefit of the remaining 0.01%).

Reply:
> This can be broken by calling CacheItem and passing it an item whose uuid field is zero
With our product I definitely had a problem like that, but I couldn't understand why and right now I'm not able to replicate it so I'll just delete it to test it, but in case it'll happen again I'll tell you

So yes, I didn't do for the performance, but I noticed that it could also improve that so I pointed that out.